exporter/kineticaexporter/factory.go

```diff
@@ -0,0 +1,115 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:generate mdatagen metadata.yaml
```

Comment:
```suggestion
```

We should only need this line in `doc.go`.

Reply:
Removed this line.